requirements.txt

```diff
@@ -7,5 +7,5 @@ setuptools
 #
 pyserial>=3.0
 future>=0.15.2
-cryptography>=2.1.4
```

Comment:
I'm afraid that this change cannot be accepted because the `nvs_flash` component relies on this version. Older versions produce the following error: `ValueError: Invalid key size (512) for AES.`